tests/php/ORM/DataObjectTest/DataListQueryCounter.php

```diff
@@ -24,10 +24,10 @@ public function __construct($dataClass)
         $this->parent = $this;
     }
 
-    public function getFinalisedQuery($queriedColumns = null)
+    public function execute()
     {
         $this->increment();
-        return parent::getFinalisedQuery($queriedColumns);
+        return parent::execute();
```

Comment:
`getFinalisedQuery()` is called for many purposes, including when calling `sql()` to generate the cache key for the new caching functionality.
`execute()` is only called when actually executing the query, which is what this was meant to be counting.